typescript/sign/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "signing",
+  "version": "0.1.0",
+  "description": "",
+  "main": "src/main.ts",
+  "scripts": {
+    "signing": "TS_NODE_PROJECT='./tsconfig.json' ts-node ./src/main.ts"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/TrueLayer/paydirect-signing-examples.git"
+  },
+  "author": "David Hayden <david.hayden@truelayer.com>",
```

Comment:
Suspicious 🕵🏻‍♂️

Reply:
I just moved the sign example, it already existed